custom_components/dreo/pydreo/constant.py

```diff
@@ -180,3 +180,12 @@ class HeaterOscillationAngles(StrEnum):
         SIXTY = "60°",
         NINETY = "90°",
         ONE_TWENTY = "120°"
+
+# Fan oscillation modes
+class FanOscillationAngles(StrEnum):
```

Comment:
Are these actually the only values this supports?  If not, we shouldn't have these here.

Reply:
Resolved, as you are right :)